index.dd

```diff
@@ -415,6 +415,7 @@ Macros:
     EXTRA_EXAMPLE=<div class="your-code-here-extra" style="display:none">$0</div>
     LAYOUT_PREFIX=
     LAYOUT_SUFFIX=
+    $(SCRIPTLOADA $(ROOT_DIR)js/platform-downloads.js, data-latest="$(LATEST)")
```

Comment:
A for SCRIPTLOAD w/ attributes, similar to DIVC for div with classes.
Just extending the existing SCRIPTLOAD runs into some annoying spacing issues without an argument.